### PR TITLE
fix: mock issue resolved for circular ref

### PR DESCRIPTION
**Pull request by SB-rohitdesai**
Addresses #[STOP-184]

**Summary**

Getting 500 Error or Sometimes might need to wait a long time before getting this error when spec having circular references

**Checklist**

- The basics
  - [x] I tested these changes manually in my local or dev environment
- Tests
  - [x] Added or updated
  - [ ] N/A
- Event Tracking
  - [ ] I added event tracking and followed the event tracking guidelines
  - [ ] N/A
- Error Reporting
  - [ ] I reported errors and followed the error reporting guidelines
  - [ ] N/A

[STOP-184]: https://smartbear.atlassian.net/browse/STOP-184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/http/src/mocker/index.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ const mock: IPrismComponents<IHttpOperation, IHttpRequest, IHttpResponse, IHttpM
   config,
 }) => {
   const payloadGenerator: PayloadGenerator = config.dynamic
-    ? partial(generate, resource, resource['__bundle__'])
+    ? partial(generate, resource, resource['__bundled__'])
```
Comment (prafullaAtSB) [inline packages/http/src/mocker/index.ts:61]: Why this has changed ?? how it was working previously?

Comment (SB-rohitdesai) [reply packages/http/src/mocker/index.ts:61]: Dereferencing was properly working with previous code (resource['__bundle__']) But getting issue in the API response. So due that I make change here.

**Review by bhaskarsontakke — APPROVED** [review]
looks good to me

**Review by prafullaAtSB — APPROVED** [review]
Good to go ahead

**Pull request closed without merging**